### ISSUE TITLE
Avoid adding own endpoints to bootstrap file

### ISSUE THREAD

**Issue by Fraser999** [maidsafe/crust#416]
If a node adds its endpoints to its own bootstrap file, on a subsequent run it connects to itself during bootstrap.

To avoid this, we should not write our own endpoints to the bootstrap file, but we should still include these in response to beacon requests as we currently do.

**Comment by inetic** [comment]
Oh, I misunderstood this one when we spoke about it yesterday. I though you wanted to not put our endpoints into the config file (not bootstrap.cache file). This one in particular is a bit tricky one because what isn't our endpoint now, may become our endpoint next time the app starts. But there should already be a [workaround](https://github.com/maidsafe/crust/pull/402/files) in place made by Vinicius as part of [this issue](https://github.com/maidsafe/crust/issues/388) (which I now think is a duplicate of this one).

TLDR; The workaround makes is so that we never put anything that is on LAN to the bootstrap cache file.

So I'm going to close this one, please reopen if needed.
